src/App/App.js

```diff
@@ -172,6 +177,13 @@ class App extends Component {
       localStorage.setItem('userFaveDecks', JSON.stringify(newFaveDecks)));
   }
 
+  removeWishListItem = (indexToRemove) => {
+    const [...newWishList] = this.state.wishList;
+    newWishList.splice(indexToRemove, 1)
+    this.setState({ wishList: newWishList },
+      localStorage.setItem('wishList', JSON.stringify(newWishList)));
+  }
+
```

Comment:
Since the `removeFaveListItem` and the `removeWishListItem` methods are so similar, I wonder if it would be possible to have a single method that does this. Maybe in addition to `indexToRemove` the method would have a parameter of `listToModify` which would be a string and everywhere `userFaveDecks` or `wishList` is could be interpolated with `listToModify`.